governance.md

```diff
@@ -84,7 +84,45 @@ The initial composition of the SC is self-defined and is made of 10 members:
 
 #### Selection
 
-The initial SC members will be responsible for defining how and when the SC membership can be expanded or renewed.
+The convention of the SC is for each company that is significantly engaged
+in the project to be represented by one or two members.
+Selection of the representatives of each company is not within scope
+of the SC.
+Membership changes can fall into one of four categories.
+Each of these processes should be initiated by the opening of a PR against
+this document, explaining the motivations of the change and introducing
+any potential new members.
+Membership changes can be approved via GitHub and do not require an SC meeting.
```

Comment:
> Membership changes can be approved via GitHub and do not require an SC meeting.

does it mean a vote is same as a PR approval?

Reply:
Good call. I guess we should have a majority of the current TSC members approving the PR to get it merged?

Reply:
Agree, that was my thinking too.

Reply:
I clarified the voting procedure slightly. For expansion, 2/3rds are required. We don't have any means to reverse expansion decisions, so it's reasonable to have a higher bar. On GH that means 2/3rds of SC members approve the PR, which is roughly equivalent to a unanimous consensus at an SC meeting with a voting quorum (2/3rds) present.

Reply:
> On GH that means 2/3rds of SC members approve the PR, which is roughly equivalent to a unanimous consensus at an SC meeting with a voting quorum (2/3rds) present.

IMHO, this is **very reasonable**, and more reasonable than a voting done in a meeting, as it gives time for the TSC members to jump in and approve (or not) the expansion, and also does **NOT** create any scenario where competitors could be blocked to join the project.